operator-wg/whitepaper/11_Related_Work.md

```diff
@@ -1,13 +1,23 @@
 **Status**: WIP | **Maintainer** : N/A | 
 
 
-## Related Work / What’s next? / Where to get help?
-**(Current) Issue: https://github.com/cncf/sig-app-delivery/issues/47**
-
+## Related Work
 Initially, Operators were introduced by a blog post on the CoreOS Blog (Philips, 2016) . This article provides a rough overview what operators are, why the concept has been developed and how they are built. The insights of this article are mainly used for the definition of operators in this document. As the blog post only provided a concise overview, additional terms as capabilities, security and additional concepts are described more in-depth in this document.
 
+Ref: Philips, B. (2016, 03 16). Introducing Operators: Putting Operational Knowledge into Software. CoreOS Blog. Retrieved 11 24, 2020, from https://coreos.com/blog/introducing-operators.html
+
 The Book “Kubernetes Operators” (Dobies & Wood, 2020) provides a comprehensive overview about operators, which problems they solve and the different methods to develop them. Definitions made in this book flowed into this document. The same applies to the Book “Kubernetes Patterns” (Ibryam, 2019), which provides more technical and conceptual insights to operators. Definitions made in these books were summarized in this document (to provide a common declaration of operators).
 
-Many documents describe capability levels of operators, which should describe the maturity of an operator.
+Ref: Dobies, J., & Wood, J. (2020). Kubernetes Operators. O'Reilly.
+
+Michael Hausenblas and Stefan Schimanski wrote a book about Programming Kubernetes, which provides deeper insights into client-go, custom resources, but also about writing operators. 
+
+Ref: Michael Hausenblas and Stefan Schimanski, Programming Kubernetes: Developing Cloud-Native Applications, First edition. (Sebastopol, CA: O’Reilly Media, 2019).
+
+Many documents describe capability levels of operators, which should describe the maturity of an operator. As there could be the case that an operator which supports all features of the highest level is not capable of some lower-level features, this document covers rather capabilities instead of capability levels. The capabilities used for defining capability levels were taken into consideration for this document. 
```

Comment:
> ...features of the highest level

I am a not sure I understand this paragraph. Could you give me an example of a high/highest level feature?

Reply:
The auto-remediation and auto-config-tuning capability of our capability model would be included of the Auto-Pilot maturity level ...

Reply:
I am not sure I understood fully what this paragraph means so I tried to re-write it to see if my thought is correct, given your explanation. If so, what do you think of expressing it this way:
```suggestion
Many documents describe capability levels (also known as maturity levels) of operators. Since there could be cases where an operator that supports all features that fall on the highest capability level but does not support some lower level features, this document chooses to cover “capabilities” rather than “capability levels”. The capabilities required for each capability level, however, are taken into consideration.
```